src/content/getting-started/secure-your-app/_index.md

```diff
@@ -1,29 +1,159 @@
 ---
+description: Learn how security is applied to your application on the Giant Swarm platform.
 title: Secure your app
-description: Learn how to secure and protect your application on the Giant Swarm platform.
 weight: 70
-last_review_date: 2024-10-08
+last_review_date: 2024-11-05
+menu:
+  principal:
+    parent: getting-started
+    identifier: getting-started-security
 owner:
   - https://github.com/orgs/giantswarm/teams/sig-docs
 user_questions:
   - How do I observe the platform metrics and logs for my application?
   - What do I need to do to observe the platform metrics and logs for my application?
 ---
 
-<Introduction>
+Giant Swarm platform is secured by default. The workload clusters are configured with security best practices in mind. The core functionality is built based on a policy enforcement engine. The `Kubernetes` project comes with [`Pod Security Standards (PSS)`](https://kubernetes.io/docs/concepts/security/pod-security-standards/) to ensure that the pods are running with the least privilege. The main problem of this technology is settings are cluster-wide and it's hard to create exceptions for specific workloads.
+
+For that reason, the platform comes with an extended policy engine, `Kyverno`. A part of enforce the common pod security standards, your clusters have additional policies which can be customized to your needs. Your developers can create exceptions for their workloads without compromising the security of the cluster.
+
+In this guide, you will learn how to make your application compliant with the secure baseline and how create policy exceptions when needed.
 
 ## Requirements
 
-<requirements>
+Before start, you need a running workload cluster. If you don't have one, please first [create a workload cluster]({{< relref "/getting-started/provision-your-first-workload-cluster" >}}).
+
+A part, you need to deploy an application, here `hello-world` application is used, to the workload cluster. For explanation on how to deploy the application please read the [previous step]({{< relref "/getting-started/install-an-application" >}}).
+
+## Step 1: Understand the security baseline
+
+By default the clusters come with a `restricted` set of policies. You can read more about which policies are part of the `restricted` set in the [upstream documentation](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted). The `restricted` set of policies are applied to all the namespaces in the cluster.
```

Comment:
```suggestion
By default, clusters come with a `restricted` set of policies. You can read more about which policies are part of the `restricted` set in the [upstream documentation](https://kubernetes.io/docs/concepts/security/pod-security-standards/#restricted). The `restricted` set of policies are applied to all the namespaces in the cluster.
```

I would like to add somewhere here some reference to our own docs:
https://docs.giantswarm.io/vintage/advanced/security/security-policy-enforcement/#default-policies